README.md

```diff
@@ -1 +1,13 @@
-# ipu-playbook
\ No newline at end of file
+# IPU-Playbook
+
+Welcome to the IPDK IPU Playbook repository! This repository is designed to provide resources, guidance and automation scripts for enabling offload features for Intel&reg; Infrastructure Processing Units (IPUs) such as Intel&reg; E2100.
```

Comment:
```suggestion
Welcome to the IPDK IPU Playbook repository! This repository is designed to provide resources, guidance, and automation scripts for enabling offload features for Intel&reg; Infrastructure Processing Units (IPUs) such as Intel&reg; E2100.
```
I would suggest breaking this into 80-character (or less) lines, and let Markdown run them together.